doc/book/connectors/index.rst

```diff
@@ -105,25 +117,56 @@ script:
     box.schema.user.grant('guest','read,write','space','examples')
     box.schema.user.grant('guest','read','space','_space')
 
+.. _index_connector_java:
+
 .. include:: __java.rst
 
+.. _index_connector_go:
+
 .. include:: __go.rst
 
+.. _index_connector_r:
+
 .. include:: __r.rst
 
+.. _index_connector_erlang:
+
 .. include:: __erlang.rst
 
+.. _index_connector_perl:
+
 .. include:: __perl.rst
 
+.. _index_connector_php:
+
 .. include:: __php.rst
 
+.. _index_connector_py:
+
 .. include:: __python.rst
 
+.. _index_connector_nodejs:
+
 .. include:: __nodejs.rst
 
+.. _index_connector_csharp:
+
 .. include:: __csharp.rst
 
+.. _index_connector_c:
+
 .. include:: __c.rst
 
+.. _index_connector_cxx:
```

Comment:
This is madness, really.
Let's make a child document rather than extending this document even more.